python/ray/tune/config_parser.py

```diff
@@ -106,6 +106,21 @@ def make_parser(parser_creator=None, **kwargs):
         default="",
         type=str,
         help="Optional URI to sync training results to (e.g. s3://bucket).")
+    parser.add_argument(
+        "--trial-string-creator",
+        default=None,
+        help="Optional creator function for the trial string, used in "
+        "generating a trial directory.")
+    parser.add_argument(
+        "--sync-cmd-tmpl",
+        default="",
+        type=str,
+        help="Optional template for syncer to run. Needs to "
+        "include replacement fields '{local_dir}' and '{remote_dir}'.")
```

Comment:
Instead of a string how about just have `sync_function` here. You can also say "btw if you pass a string it will be used by this default function".

Reply:
done